File: src/angular-product-app/src/app/models/payment-card.ts
export class PaymentCard {
  holderName: String;
  cardNumber: String;
  expirationMonth: number;
  expirationYear: number;
  cvv: String;


  constructor(holderName: String, cardNumber: String, expirationMonth: number, expirationYear: number, cvv: String) {
    this.holderName = holderName;
    this.cardNumber = cardNumber;
    this.expirationMonth = expirationMonth;
    this.expirationYear = expirationYear;
    this.cvv = cvv;
  }
}

File: src/main/java/com/yaroslavpiskachov/model/PaymentCard.java
package com.yaroslavpiskachov.model;

public class PaymentCard {
    private String holderName;
    private String cardNumber;
    private Integer expirationMonth;
    private Integer expirationYear;
    private Integer cvv;

    public String getHolderName() {
        return holderName;
    }

    public void setHolderName(String holderName) {
        this.holderName = holderName;
    }

    public String getCardNumber() {
        return cardNumber;
    }

    public void setCardNumber(String cardNumber) {
        this.cardNumber = cardNumber;
    }

    public Integer getExpirationMonth() {
        return expirationMonth;
    }

    public void setExpirationMonth(Integer expirationMonth) {
        this.expirationMonth = expirationMonth;
    }

    public Integer getExpirationYear() {
        return expirationYear;
    }

    public void setExpirationYear(Integer expirationYear) {
        this.expirationYear = expirationYear;
    }

    public Integer getCvv() {
        return cvv;
    }

    public void setCvv(Integer cvv) {
        this.cvv = cvv;
    }
}
